keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/README.md

```diff
@@ -30,21 +30,41 @@ This sample should work together with [azure-spring-boot-sample-keyvault-certifi
 
     Then you will get
     ```text
-    Response from "https://localhost:8443/": Hello World
+    Response from restTemplate tls "https://localhost:8443/": Hello World
     ```
 
-#### Using mTLS with service principal created via App registration.
-1. In the sample `ApplicationConfiguration.class`, change the `self-signed` to your certificate alias.
-    <!-- embedme ../azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side/src/main/java/com/azure/spring/security/keyvault/certificates/sample/client/side/SampleApplicationConfiguration.java#L70-L75 -->
+#### Using TLS with service principal and WebClient
+
+1. Add below properties to the `application.yml` of client side:
+
+   ```yaml
+   spring:
+     profiles:
+       active: webclient
```

Comment:
Use profile make this sample harder to use. How about removing profile?

Reply:
I don't think so, from my point of view, while I test the sample again and agaiin, I prefer to use profile to choose using webclient or resttemplate.

Reply:
How about enabling `webClient` and `restTemplate` at the same time? We already use different endpoint for them.

Reply:
It will double the time to start the sample, especially when the keyvault has more than one certificate.

Reply:
I refactored the code, now for the 4 beans, restTemplateWithTLS, restTemplateWithMTLS, webClientWithTLS, webClientWithMTLS it will only load KeyVaultKeyStore once.